### PR TITLE
IDEX-3805: replace HttpJsonHelper with HttpJsonRequest

### PR DESCRIPTION
**Pull request by garagatyi**

### CODE REVIEW

**Comment by skabashnyuk** [comment]
ok

**Comment by vzhukovskii** [comment]
What the purpose of standalone module with single file?

**Comment by garagatyi** [comment]
For now it is only one file. Later we can add more

File: commons/che-core-commons-test/pom.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    Contributors:
+      Codenvy, S.A. - initial API and implementation
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <artifactId>che-core-commons-parent</artifactId>
+        <groupId>org.eclipse.che.core</groupId>
+        <version>4.0.0-beta-6-SNAPSHOT</version>
```
Comment (vzhukovskii) [inline commons/che-core-commons-test/pom.xml]: wrong version

**Comment by vzhukovskii** [comment]
You can place SelfReturningAnswer into che-core-commons-lang. No?

**Comment by skabashnyuk** [comment]
no

File: commons/che-core-commons-test/pom.xml (hunk 1 above)
Comment (garagatyi) [inline commons/che-core-commons-test/pom.xml]: I just updated the PR

**Pull request merged**